server/loader.py

```diff
@@ -168,8 +55,27 @@ def validate_settings(event):
 
 
 def load(info):
+    # Load the mako template for Minerva and serve it as the root document.
+    minerva_mako = os.path.join(os.path.dirname(__file__), "minerva.mako")
+    minerva_webroot = Webroot(minerva_mako)
+
+    # Load resources from all enabled plugins, which allows Minerva
+    # to be extended by other Girder plugins.
+    plugins = ModelImporter.model('setting').get(SettingKey.PLUGINS_ENABLED,
+                                                 ())
+    plugins = list(plugins)
+    if 'minerva' in plugins:
+        plugins.remove('minerva')
+    vars = {
+        'plugins': plugins,
+        'apiRoot': '/api/v1',
+        'staticRoot': '/static',
+        'title': 'Minerva'
+    }
+    minerva_webroot.updateHtmlVars(vars)
```

Comment:
> Will do. Can you give an example of such a scenario so I can understand the issue better?

[This section](http://girder.readthedocs.org/en/latest/developer-cookbook.html#serving-a-custom-app-from-the-server-root) of the docs discusses moving the girder webroot and the API root within cherrypy, which is another reason the API might not be at `/api/v1`. The static root may point, for instance, to a CDN, or get moved by a plugin for whatever reason.

Reading these vars from the core template is probably a good solution for now.